### PR TITLE
warn when trying to enter json, but the json is invalid

### PR DESCRIPTION
**Pull request by grosser**
@jonmoter @irwaters 

we will dump some json into secrets ... validation will be nice once they get complicated

### CODE REVIEW

**Comment by grosser** [comment]
screen shot 2016-08-01 at 1 54 45 pm

File: app/views/admin/secrets/edit.html.erb [hunk 1]
```diff
@@ -107,4 +108,25 @@ $(function () {
...
+
+// when it looks like the user enters json, tell them when it is invalid
+// when they correct it, clear the warning
+$(function () {
+  var $value = $("#secret_value");
+  var $warn = $("#value_json_warning");
+
+  $value.keyup(function () {
+    var val = $value.val();
+    if(/^\s*\{[\s\S]*\}\s*$/.test(val)){
```
Comment (jonmoter) [inline app/views/admin/secrets/edit.html.erb]: Catch arrays as well as hashes as the outermost characters.

**Comment by grosser** [comment]
fixed

**Comment by jonmoter** [comment]
👍 on 💚

**Pull request merged**